examples/CivisML_parallel_training.ipynb

```diff
@@ -149,11 +150,11 @@
     "COL_NAMES = ['sample_id', 'clump_thickness', 'uniformity_cell_size', 'uniformity_cell_shape',\n",
     "            'marginal_adhesion', 'epithelial_cell_size', 'bare_nuclei', 'bland_chromatin',\n",
     "            'normal_nucleoli', 'mitoses', 'is_cancer']\n",
-    "df = pd.read_csv(\"breast_cancer_data.csv\", names=COL_NAMES, na_values='?')\n",
+    "df = pd.read_csv(\"breast_cancer_data.csv\", names=COL_NAMES, na_values=np.nan)\n",
```

Comment:
This line should stay the same. The `na_values` argument tells `pandas` which strings represent missing data. All of them get encoded as `np.nan`. We still want any "?" strings present in the CSV to be treated as missing data.

https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html#pandas-read-csv